lib/tasks/rubocop.rake

```diff
@@ -13,6 +13,16 @@
 #
 #++
 
+def rubocop_bin
+  return @rubocop_bin if @rubocop_bin
+  version = system("grep 'rubocop-0.71.0' .rubocop.yml") ? "0.71.0" : "0.41.2"
```

Comment:
Maybe we should allow overriding the detection via environment variable (like `RUBOCOP_BIN`), if you install the Rubocop using `gem install` instead of RPM (for whatever reason) then the update-alternatives is not configured for it.